debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java

```diff
@@ -171,6 +275,7 @@ private void checkForSwitchOperationAndHandle(String operation) {
             LOGGER.info("{} processing complete. Exiting...", operation);
             shutDown = true; // to ensure that no event gets written after switch operation.
         }
+        releaseLockFile();
```

Comment:
wouldn't system.Exit automatically handle it because of the exit hook?

Reply:
yes it will be handled by the shutdown hook, but I thought initially that when we are calling `system.exit` we can release the lock file on our own and not rely on shutdown hooks as these can sometimes not run properly and so on but now when I think it is okay to depend on shutdown hook as that's the purpose of it and in go layer as well we rely on exit handler for all work to be done during exits/terminations and we don't call them where ever we call os.exit etc.. so removed it from the both places.